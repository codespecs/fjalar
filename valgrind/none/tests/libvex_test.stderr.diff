--- libvex_test.stderr.exp	2022-08-03 11:30:53.706309390 -0700
+++ libvex_test.stderr.out	2022-08-04 11:59:00.940901618 -0700
@@ -0,0 +1,2 @@
+valgrind: You cannot run './libvex_test' directly.
+valgrind: You should use $prefix/bin/valgrind.
