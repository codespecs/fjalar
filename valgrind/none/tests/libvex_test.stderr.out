valgrind: You cannot run './libvex_test' directly.
valgrind: You should use $prefix/bin/valgrind.
